README.md

```diff
@@ -63,7 +63,7 @@ Please find the example of provisioning Grafana instance with VictoriaLogs datas
    ``` bash
    ver=$(curl -s https://api.github.com/repos/VictoriaMetrics/victorialogs-datasource/releases/latest | grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+' | head -1)
    curl -L https://github.com/VictoriaMetrics/victorialogs-datasource/releases/download/$ver/victorialogs-datasource-$ver.tar.gz -o plugin.tar.gz
-   tar -xf plugin.tar.gz -C ./victorialogs-datasource
+   tar -xf plugin.tar.gz -C ./
```

Comment:
if we remove the folder in the next step we will not be able to correctly align the folder in the docker-compose file

```
  version: '3.0'
  services:
     grafana:
        container_name: 'victorialogs-datasource'
        build:
           context: ./.config
           args:
              grafana_version: ${GRAFANA_VERSION:-10.0.3}
        ports:
           - 3000:3000/tcp
        volumes:
           - ./victorialogs-datasource:/var/lib/grafana/plugins/victorialogs-datasource
           - ./provisioning:/etc/grafana/provisioning
```